Tensile/KernelWriter.py

```diff
@@ -2202,8 +2202,8 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
             kl.append(self.macIter(kernel, 0, tailLoopInnerUnroll, True))
 
         kl.append(self.closeLoop(kernel, -1, True, uDu if kernel["DepthULdsDivisor"]>1 else None))
-    if kernel["DepthULdsDivisor"]>1:
-      kl.append(self.closeLoop(kernel, -1, None, emitEndLabelOnly=True))
+    # always emit the skip-tail-loop label
+    kl.append(self.closeLoop(kernel, -1, None, emitEndLabelOnly=True))
```

Comment:
Emit a SkipTailLoop label which skip any code of tail-loop at the end of tail loop. No matter we have DULDS or not.